spring-cloud-stream-core-docs/src/main/asciidoc/preface.adoc

```diff
@@ -1,3 +1,19 @@
+== A Brief History of Spring's Data Integration Journey
+
+Spring's journey on Data Integration started with https://projects.spring.io/spring-integration/[Spring Integration]. With its programming model, it provided a consistent developer experience to build applications that can embrace http://www.enterpriseintegrationpatterns.com/[Enterprise Integration Patterns] to connect with external systems such as databases, message brokers, and among others.
+
+Fast forward to cloud-era, where microservices were prominent in the enterprise setting. https://projects.spring.io/spring-boot/[Spring Boot] transformed the way how developers built Applications. With Spring's programming model and the runtime responsibilities handled by Spring Boot, it became seamless to develop standalone production grade Spring-based microservices.
+
+To extend this to Data Integration workloads, Spring Integration and Spring Boot were put together into a new project. Spring Cloud Stream was born.
+
+With Spring Cloud Stream, developers can:
```

Comment:
Add a period at the end of each list item. Because some of them are sentences, they all have to be punctuated that way.